api/app.ts

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import { getCalendarEntries } from './controllers/calendarEntry.controller'
+
+const app = express();
+const port = 3000;
```

Comment:
Is 3000 the port for the backend server. I'm noticing we also have the front end on port 3000. How does it know the difference between the two?

Reply:
Good catch - we'll update this.